EleventhLesson/task1/Main.java

```diff
@@ -0,0 +1,24 @@
+package task1;
+
+public class Main {
+    public static void main(String[] args) {
+        int[] arr = new int[10];
+        for (int i = 0; i < arr.length; i++) {
+            arr[i] = Utils.enterNumber();
+        }
+        Thread myThreadMax = new Thread(new MyThreadMax(arr));
+        Thread myThreadMin = new Thread(new MyThreadMin(arr));
+        myThreadMax.start();
```

Comment:
Чтобы программа отработала быстрее лучше запустить сразу два потока и после дожидаться их окончания. В этом и смысл распараллеливания задачи
```suggestion
        myThreadMax.start();
        myThreadMin.start();
        try {
            myThreadMax.join();
            myThreadMin.join();
        } catch (InterruptedException e) {
            e.printStackTrace();
        }
```